src/Sdk/DTWebApi/WebApi/Exceptions.cs

```diff
@@ -2459,6 +2459,25 @@ private InvalidContinuationTokenException(SerializationInfo info, StreamingConte
         }
     }
 
+    [Serializable]
+    public class UnresolvableActionDownloadInfoException : DistributedTaskException
```

Comment:
I don't see any `new UnresolvableActionDownloadInfoException` calls being added in this PR. Does this exception type get used automatically somehow?

Reply:
Is this auto-resolved in the task client library and this is just a missed port?

Reply:
Yes, from my local testing this is auto resolved once I ported the exception type.